### ISSUE TITLE
More informative `-help` message for `joern`

### ISSUE THREAD

**Issue by fabsx00** [ShiftLeftSecurity/joern#173]
When running `joern -help`, we end up mainly with information about the JVM as opposed to information about `--script` and `--params` as one would except. Let's make sure that when a user types `joern -h` or `joern --help`, a cleaned up info text appears.

**Comment by GlassAndOneHalf** [comment]
See https://github.com/ShiftLeftSecurity/codepropertygraph/pull/487

Once this is merged into `codepropertygraph` we can simply bump the version used in Joern and the help will display as expected.

**Comment by GlassAndOneHalf** [comment]
Fixed by https://github.com/ShiftLeftSecurity/joern/pull/181
